ui/shared/utils/stringUtils.js

```diff
@@ -14,7 +11,4 @@ export const toCamelcase = s => (s || '').split(' ').map(
   (word, i) => word && (i > 0 ? word[0].toUpperCase() : word[0].toLowerCase()) + word.slice(1),
 ).join('')
 
-const TEXT_RENDERERS = Object.keys(markdownRenderers).reduce((acc, k) => ({ ...acc, [k]: markdownRenderers.text }), {})
-
-export const stripMarkdown =
-  s => ReactDOMServer.renderToStaticMarkup(<ReactMarkdown renderers={TEXT_RENDERERS}>{s || ''}</ReactMarkdown>).trim()
+export const stripMarkdown = s => removeMd(s || '').replaceAll('\n', ' ')
```

Comment:
use a lightweight library for this instead of the very heavy libraries, which are changed to be lazily loaded in the one other place that needs them